server/views/applications/pages/oasys-import/rosh-summary.njk

```diff
@@ -0,0 +1,122 @@
+{%- from "moj/components/sub-navigation/macro.njk" import mojSubNavigation -%}
+{% from "../../components/riskWidgets/macro.njk" import widgets %}
+
+{% extends "../layout.njk" %}
+
+{% set columnClasses = "govuk-grid-column-full" %}
+
+{% block questions %}
+  <div class="govuk-grid-row">
+    <div>
+      <h1 class="govuk-heading-l">{{page.title}}</h1>
+      <p>Imported from OASys 23 October 2022</p>
+    </div>
+
+    {{ mojSubNavigation({
+        label: 'Sub navigation',
+        attributes: {
+          'role': 'tablist'
+        },
+        items: [{
+          text: 'RoSH summary',
+          href: '#roshSummary',
+          attributes: {
+            'aria-controls': 'roshSummary',
+            'id': 'roshSummaryTab',
```

Comment:
Is the differing id and aria-controls intentional here?

Reply:
No, I've fixed this